client/src/components/Splash.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class Splash extends Component {
```

Comment:
do you think we can make this a functional component? or do we need to emit an action when someone clicks on "Add to Slack" (to redirect to another page?). In the latter case, we might need to make this a redux container